### DISCUSSION TITLE
Why does Core Max go down?  It should only go up to record the Max??

### DISCUSSION THREAD

**Discussion by Snacko** [Rem0o/FanControl.Releases#1797] [General]
I'm on an Intel i9-10850k and I noticed that the Core Max goes both up and down?  
I expected it to display the **Max Core Temp.**  Is that what it is supposed to do, or am I not understanding something??

**Comment by Rem0o** [comment]
Core max is the highest temperature amongst all of your cores at any given time.

**Comment by Snacko** [comment]
Ok, thanks!
